Country.py

```diff
@@ -0,0 +1,270 @@
+        self.color = color
+        self.radius = radius
+        self.infected_pop = infected_pop
+        self.infected_rate = infected_rate
+        self.max_pop = max_pop
+        self.dead_pop = dead_pop
+        self.death_rate = death_rate
+        self.airborne_rate = airborne_rate
+
+    def infected_ratio(self):
+        """
+        The ratio of infection (population of infected people / toal population of a country)
+        """
+        if self.max_pop != 0:
+            return int(self.infected_pop) / self.max_pop
+        else:
+            return 1
+
+    def death(self):
+        """
+        A part of the infected population would be passed away.
+        Then the infected population and maximum population will be reduced as many as the number of people death.
+        """
+        death_pop = 0
+        alive_pop = self.max_pop
+        if self.infected_ratio() > 0.10:
+            if self.infected_pop > 10:
+                death_pop = int(self.death_rate*self.infected_pop*(random.random()/15))
+            else:
+                if self.max_pop >= 1:
+                    death_pop = 1
+                else:
+                    death_pop = 0
+
+        self.infected_pop = self.infected_pop - death_pop
+        self.max_pop -= death_pop
+        self.dead_pop += death_pop
+
+    def step(self):
+        """
+        If infection starts (infect_pop changed into unity),
+        then the infection starts with the certain rate
+        """
+        if self.infected_pop < self.max_pop:
+            self.infected_pop = self.infected_pop * self.infected_rate
+            if self.infected_pop >= self.max_pop:
+                self.infected_pop = self.max_pop
+
+        """return integer part of infected population"""
+        # return int(self.infected_pop)
+
+        """return probability"""
+        return self.infected_ratio()
+
+
+    def draw(self):
+        """ draws the location of the country as a circle """
+        pygame.draw.circle(screen, self.color, (int(self.x), int(self.y)),
+                           self.radius)
+
+    def contains_pt(self, pt):
+        """ countains points function to see if mouseclicks are
+        within the circles of the country """
+        x, y = pt
+        if not self.x - self.radius < x < self.x + self.radius:
+            return False
+        if not self.y - self.radius < y < self.y + self.radius:
+            return False
+        return True
+
+    def propagation(self, other):
+        """
+        The pathogen can propagate to other countries with certain probability.
+        """
+        if self.infected_pop >= 1 and other.infected_pop == 0:
+            if random.random() <= self.infected_ratio()/10:
+                other.infected_pop = 1
+
+
+background_color = (255,255,255)
+width, height = 640, 480
+
+screen = pygame.display.set_mode((width, height))
+pygame.display.set_caption('Plague Simulation')
+screen.fill(background_color)
+
+intro = True
+
+while intro:
```

Comment:
It would be nice to put the game intro, main loop, and ending is separate functions that you can call at the appropriate times. This helps with code organization.